core/types/confidential.go

```diff
@@ -17,6 +17,9 @@ type ConfidentialComputeRecord struct {
 	KettleAddress          common.Address
 	ConfidentialInputsHash common.Hash
 
+	// Envelope signals whether this CCR was signed using EIP-712
+	Envelope bool
```

Comment:
nit: this feels not very descriptive

Reply:
Agree, what about `EIP712SigningSchema: bool`?

Reply:
Since you two may not fully grasp the context, please let me know if my comments are off the mark.

This is a boolean, so I feel that using a format like `isXxx` would improve readability. What do you think? 
For example, how about simply using `isEIP712: bool` or something similar?

Reply:
I like `IsEIP712` or `SignedWithEIP712`